### PR TITLE
OKRSPY24-281 | @rebeccahongsf | build slideshow carousel 

### PR DESCRIPTION
**Pull request by rebeccahongsf**
# READY FOR REVIEW

# Summary
- build slideshow carousel

# Review By (Date)
- When possible

# Review Tasks

## Setup tasks and/or behavior to test

1. Navigate to https://deploy-preview-6--sws-year-in-review.netlify.app/
2. Verify that the slideshow functions
3. Review code

### CODE REVIEW

File: app/globals.css [hunk 1]
```diff
@@ -1,3 +1,39 @@
...
+  @apply rs-py-8 px-0;
+}
+
+.slick-slide > div {
+  @apply rs-mx-1;
+}
+
+@media (min-width: 992px) {
+  .center .slick-slide.slick-active.slick-current {
+    @apply z-50 transition-all duration-[400ms] ease-[ease-in-out] scale-[1.5] opacity-100 relative;
```
Comment (rebeccahongsf) [inline app/globals.css:19]: @yvonnetangsu any thoughts or feedback on how the scaling of the center slide? 🤔 
The scale increase impacts the caption sizing as well.

Comment (yvonnetangsu) [reply app/globals.css:19]: @rebeccahongsf I didn't look at the code so just answering theoretically. Is it possible to have a wrapper around the image and caption, then you scale the wrapper, and just have the image inside be full width of the wrapper?

Comment (yvonnetangsu) [reply app/globals.css:19]: I almost feel like we should only show the caption when the slide is in the middle 🤔

**Comment by yvonnetangsu** [comment]
@rebeccahongsf The arrow buttons seem to behave backward for me 😄

**Comment by rebeccahongsf** [comment]
> @rebeccahongsf The arrow buttons seem to behave backward for me 😄

Ah! 😮‍💨 Great catch, @yvonnetangsu! It should be corrected now

**Comment by yvonnetangsu** [comment]
And could we have some focus states for the dots please? 😄 Also, knowing SODA, I think they might think the focus order should be on the arrow button first, then the dots since the arrows are higher up. I suggest you schedule an office hour and have them take a look at this. There are some things that even I'm not sure about since some of the rules are not clear, e.g., do we need to support tabbing from one slide to another even though they aren't focusable elements? 🤔

**Comment by yvonnetangsu** [comment]
Annnnnd if we could have the arrow itself be a solid black color (instead of the transparency) that'd be awesome. So we can be sure it's accessible no matter what's showing underneath 👍🏼

**Pull request merged**